lib/clientv3/node.go

```diff
@@ -115,6 +115,29 @@ func (r nodes) Delete(ctx context.Context, name string, opts options.DeleteOptio
 			}
 		}
 	}
+
+	// Add in tunnel addresses if they exist for the node.
+	n, err := r.client.Nodes().Get(ctx, name, options.GetOptions{})
+	if err != nil {
+		return nil, err
```

Comment:
What if the error is a "not found".  In that case, we might still want to clean up the IPs.